tests/test_types.py

```diff
@@ -0,0 +1,68 @@
+"""
+:Description: Unit tests for the types module
+"""
+
+from __future__ import annotations
+
+import pytest
+
+from conda_recipe_manager.types import MessageCategory, MessageTable
+
+
+@pytest.mark.parametrize(
+    "message,content,output",
+    [
+        (MessageCategory.WARNING, "Test warning message", "0 errors and 1 warning were found."),
+        (MessageCategory.ERROR, "Test error message", "1 error and 0 warnings were found."),
+    ],
+)
```

Comment:
@schuylermartin45 is there an effective way that I could add both a warning AND an error via parametrization and get the `"1 error and 1 warning were found."` output? I tried several things but kept getting a `TypeError` with too many arguments passed to `MessageTable.add_message()`.

Reply:
Although I normally advise not to put for loops in tests, in this case, I think you could combine `message` and `content` into a list of tuples to make it easier to develop test cases:
```

([(MessageCategory.WARNING, "Test warning message")], "0 errors and 1 warning were found."),
([(MessageCategory.ERROR, "Test error message"), (MessageCategory.WARNING, "Test warning message")], "1 error and 1 warning were found."),
```